eframe/src/native/epi_integration.rs

```diff
@@ -264,6 +264,14 @@ impl EpiIntegration {
         full_output
     }
 
+    pub fn post_rendering(&mut self, app: &mut dyn epi::App) {
+        let pixels_per_point = self.frame.info().native_pixels_per_point.unwrap_or(1.0);
+        let screen_size = self.egui_ctx.input().screen_rect.size() * pixels_per_point;
+        let screen_size_px = [screen_size.x as u32, screen_size.y as u32];
```

Comment:
There's a subtle bug here - you are using the native_pixels_per_point, but  `InputState::screen_rect` is based on the current `pixels_per_point` which is customizable. To get the pixel size of the display it is better to ask the `winit::Window` directly with https://docs.rs/winit/latest/winit/window/struct.Window.html#method.inner_size